lib/sensu-mutator.rb

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env ruby
+#
+# Sensu-mutator
+# ===
+#
+# DESCRIPTION:
+#   Base mutator class.  All you need to do is extend this class and implement a
+#   #mutate function.
+# 
+#  trival case as described https://sensuapp.org/docs/latest/mutators#example-mutator-plugin:
```

Comment:
Mind formatting this slightly better and adding a bit more documentation? Maybe describe the advantages of using this class opposed to the old approach.